Update code.rst (#2310)
arbor-sim/arbor: doc/contrib/code.rst

   # Install the formatter if not present
   pip install black

>    # Automatically apply style to a certain file. If unsure what this does read on.

Reviewer: ```suggestion
   # Automatically apply style to all Python files. If unsure what this does read on.
```

>    black . scripts/arbor/build-catalogue.in [check this line please]

Reviewer: ```suggestion
   black .
```

For example, if you use ``std::vector<int>`` in a file, make sure to
have ``#include <vector>`` at the top of the source file.

Relying on headers being included elsewhere can lead to portability

> problems, for example, on OS X you have to ``#include <cmath>`` for some

Reviewer: ```suggestion
problems, for example, on MacOS you have to ``#include <cmath>`` for some
```
